dist/tools/sliptty/start_network

```diff
@@ -0,0 +1,104 @@
+#! /bin/sh
+
+SLIPPY_DIR="$(cd "$(dirname "$0")" && pwd -P)"
+UHCPD="$(cd "${SLIPPY_DIR}/../uhcpd/bin" && pwd -P)/uhcpd"
+TUN=sl0
+UHCPD_PID=
+CREATED_IFACE=0
+
+unsupported_plattform() {
+    echo "unsupported platform" >&2
+    echo "(currently supported \`uname -s\` 'Darvin' and 'Linux')" >&2
+}
+
+case "$(uname -s)" in
+    Darvin)
+        PLATFORM="OSX";;
+    Linux)
+        PLATFORM="Linux";;
+    *)
+        unsupported_platform
+        exit 1
+        ;;
+esac
+
+create_tun() {
+    case "${PLATFORM}" in
+        Linux)
+            if ! ip link show ${TUN} 2>&1 > /dev/null; then
+                sudo ip tuntap add ${TUN} mode tun user ${USER} || exit 1
+                CREATED_IFACE=1
+            fi
+            sudo sysctl -w net.ipv6.conf.${TUN}.forwarding=1
+            sudo sysctl -w net.ipv6.conf.${TUN}.accept_ra=1
```

Comment:
accept_ra should be set to 2 if you want the host to accept routing advertisements when it is configured as a router (forwarding=1)

More info:
https://www.kernel.org/doc/Documentation/networking/ip-sysctl.txt
http://strugglers.net/~andy/blog/2011/09/04/linux-ipv6-router-advertisements-and-forwarding/

Reply:
Fixed